loom/common/utils/humanFormat.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * ===========================================================================
+ * Loom SDK
+ * Copyright 2011, 2012, 2013
+ * The Game Engine Company, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ===========================================================================
+ */
+
+#ifndef _HUMAN_FORMAT_H_
+#define _HUMAN_FORMAT_H_
+
+#include "math.h"
+#include "loom/common/utils/utString.h"
+
+utString humanFileSize(long bytes)
```

Comment:
It's based on me not finding any obvious small lib for this and making one. Wanted to just copy something like http://programanddesign.com/cpp/human-readable-file-size-in-c/

But mine is nicer :smirk_cat: